test/docker-e2e/dockerchain/config.go

```diff
@@ -0,0 +1,90 @@
+package dockerchain
```

Comment:
analogous to testnode, embeds the testnode.Config and extends it with Docker specific config.